src/Compilers/CSharp/Portable/Binder/Binder_Operators.cs

```diff
@@ -1422,6 +1422,12 @@ private bool IsTypelessExpressionAllowedInBinaryOperator(BinaryOperatorKind kind
 
             result.Free();
             return possiblyBest;
+
+            bool isNuint(TypeSymbol type)
```

Comment:
>isNuint(TypeSymbol type) [](http://example.com/codeflow?start=17&length=24)

Was there a reason to introduce this local function rather than leave the code inline?

Reply:
Felt more readable